src/ui/server/api/keystone/get-ordnance-survey-address/index.ts

```diff
@@ -0,0 +1,40 @@
+import { ApolloResponse } from '../../../../types';
+import apollo from '../../../graphql/apollo';
+import ordnanceSurveyAddressQuery from '../../../graphql/queries/ordnance-survey-address';
+
+/**
+ * keystone GraphQL API call for ordnance survey address
+ * returns address or errors
+ * @param {String} postcode
+ * @param {String} houseNumber
+ */
+const getOrdnanceSurveyAddress = async (postcode: string, houseNumber: string) => {
+  try {
```

Comment:
Can we add a `console.info` and update the other logs, some have uncapitalized "ordnance survey"